src/components/ComponentPages/SettingsUI/index.tsx

```diff
@@ -48,13 +45,17 @@ export default function SettingsUI() {
               <Card>
                 <Button size="large" className="createTopicBtn">
                   <Icon
-                    component={() => <Image alt="adOne" src={createTopic} />}
+                    component={() => <Image alt="adOne" src={'/images/topic-icn.svg'} width={19}
+                      height={22} />}
                   />
                   Create New Topic
                 </Button>
               </Card>
               <div className="siteAds">
-                <Image alt="adOne" src={'/images/image11.jpg'} />
+                <Image alt="adOne"
+                  src={'/images/image11.jpg'}
+                  width={200}
```

Comment:
Can we make the height and width variable dynamic or get it from config or any other global variable

Reply:
It is a static image for now. when there will be dynamic ads, then it will be removed from here